src/Microsoft.ML.FastTree/TreeEnsembleFeaturizer.cs

```diff
@@ -390,9 +428,15 @@ public TreeEnsembleFeaturizerBindableMapper(IHostEnvironment env, ModelLoadConte
 
             // *** Binary format ***
             // ensemble
+            // string: treesColumnName
+            // string: leavesColumnName
+            // string: pathsColumnName
 
             ctx.LoadModel<TreeEnsembleModelParameters, SignatureLoadModel>(env, out _ensemble, "Ensemble");
             _totalLeafCount = CountLeaves(_ensemble);
+            _treesColumnName = ctx.LoadStringOrNull();
```

Comment:
Do you need to check if the model has the correct version before calling `LoadStringOrNull`?

Reply:
Sure. Added. Thanks.